### ISSUE TITLE
Cannot mount /usr/

### ISSUE THREAD

**Issue by ggmaster-alt** [canonical/steam-snap#389]
### Ensure there isn't an existing issue for this and check the wiki

- [X] This issue is not a duplicate and I have checked the wiki.

### Current Behavior

everytime i try to open steam it cycles then runs into files that it doesnt have access to in appamor permissions, how do i fix these issues it didnt happen before i asume that its a steam updated that appamor hasnt agjusted for. 
![Screenshot from 2024-06-03 14-45-49](https://github.com/canonical/steam-snap/assets/65167267/3c033a82-97f5-4ba3-9995-87a8aba3b2b0)
![Screenshot from 2024-06-03 14-45-57](https://github.com/canonical/steam-snap/assets/65167267/b5bb8fad-98b8-4875-9e44-2ac09dd86171)

### Expected Behavior

steam should just open.

### Steps To Reproduce

1. open steam
2. a window flickers black
3. close steam as it fails to open

### Environment

```yaml
os_release:
    name:               "Ubuntu"
    version:            "22.04.4 LTS (Jammy Jellyfish)"
snap_info:
    steam_revision:     191
    snapd_revision:     21759
lspci:
    12:00.0:            Advanced Micro Devices, Inc. [AMD/ATI] Navi 23 [Radeon RX 6600/6600 XT/6600M] (rev c1)
    30:00.0:            Advanced Micro Devices, Inc. [AMD/ATI] Cezanne (rev c9)
glxinfo:
    gpu:                AMD Radeon RX 6600 XT (radeonsi, navi23, LLVM 15.0.7, DRM 3.42, 5.15.0-107-generic)
    gpu_version:        4.6 (Core Profile) Mesa 24.0.3 - kisak-mesa PPA
lscpu:
    model_name:         AMD Ryzen 5 5600G with Radeon Graphics
xdg_current_desktop:    ubuntu:GNOME
desktop_session:        ubuntu
```

### gaming-graphics-core22 version

kisak-fresh (default)

**Comment by ggmaster-alt** [comment]
found a error with my system, because ive riced it, steam will be trying to access files that are in other files it doesnt have access to.

**Comment by ashuntu** [comment]
We are reworking the app armor profiles (in [snapd](https://github.com/snapcore/snapd)) for the Steam snap pretty heavily this cycle, so at the moment this will remain unresolved until that work is complete, but should be resolved with those changes.

**Comment by zyga** [comment]
This has been fixed in snapd, and will be released in snapd 2.64.

You can give it a try with:
```
sudo snap refresh snapd --edge
```

To go back to a stable release of snapd please run:
```
sudo snap refresh snapd --stable
```

Some changes may need a reboot to apply (not really but easier than precise description).
